### PR TITLE
Suport for Public Path / Storage Path / Private Path

### PR DESCRIPTION
**Pull request by hotrungnhan**
Closes #???

##  ✋ Problem
Currently it's only Path args is accepted .

## ✔️ Solution
Treat public / private/storage path as Fcl Path
add tests to cover this use case
## 💡 Acceptance Criteria
it should be possible to pass Path arguments
new tests added and passing for all basic cases
______

For contributor use:

- [ ] Targeted PR against `master` branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work
- [ ] Code follows the [standards mentioned here](https://github.com/onflow/flow-js-testing/blob/master/CONTRIBUTING.md#styleguides)
- [ ] Updated relevant documentation 
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

**Comment by hotrungnhan** [comment]
@MaxStalker this PR will accept cadence Args Path such as Path, Public Path, PrivatePath, StoragePath. 
### Unit test is included

File: src/args.js [hunk 1]
```diff
@@ -115,7 +115,9 @@ export const resolveType = (type) => {
         const dictionaryType = { key: resolveType(key), value: resolveType(value) };
         return t.Dictionary(dictionaryType);
       }
-
+      case isPath(type): {
+        return t.Path;
+      }
```
Comment (MaxStalker) [inline src/args.js:120]:
```suggestion

      case isPath(type): {
        return t.Path;
      }
      
```

Comment (MaxStalker) [reply src/args.js:120]: Just a bit more space around the case :)

Comment (hotrungnhan) [reply src/args.js:120]: sure .

Comment (MaxStalker) [reply src/args.js:120]: > sure .

Oh, come on :)
Not trying to be picky 😛 
It's just one of the things I would personally tinker with 😅

**Review by jribbink — APPROVED** [review]

**Pull request merged**